src/components/Comments/ReviewComments.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import { getCommentsOnReview } from '../../utils/api'
+
+const ReviewComments = () => {
+  const [reviewComments, setReviewComments] = useState([])
+  const [isLoading, setIsLoading] = useState(true)
+  const { review_id } = useParams()
+
+  useEffect(() => {
+    setIsLoading(true)
+    getCommentsOnReview(review_id).then((reviewCommentData) => {
+      setReviewComments(reviewCommentData)
+      setIsLoading(false)
+    })
+  }, [review_id])
+
+  return isLoading ? (
+    <h2>ReviewComments: Loading Animation Goes Here</h2>
+  ) : (
+    <section className="comment-container">
+      <h2 className="comments-title">Comments:</h2>
+      <ul className="comment-list">
+        {reviewComments.map((comment) => {
```

Comment:
This map currently means that if there are no review comments, the reviewPage component breaks (see articles/14 for example)
Put some logic in to check that reviewComments isn't empty prior to your map to protect against this